chartfx-chart/src/test/java/de/gsi/chart/plugins/ReproduceTableViewerBug.java

```diff
@@ -0,0 +1,137 @@
+package de.gsi.chart.plugins;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.stage.Stage;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.gsi.chart.XYChart;
+import de.gsi.chart.plugins.TableViewer;
+import de.gsi.dataset.DataSetError;
+import de.gsi.dataset.testdata.spi.CosineFunction;
+import de.gsi.dataset.testdata.spi.SineFunction;
+import de.gsi.dataset.utils.ProcessingProfiler;
+
+/**
+ * Reproduces the NPE bug in the table viewer.
+ * It seems to trigger <a href="https://bugs.openjdk.java.net/browse/JDK-8217953">JDK-8217953</a>.
+ * 
+ * @author Alexander Krimm
+ */
+public class ReproduceTableViewerBug extends Application {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ReproduceTableViewerBug.class);
+    private static final String STOP_TIMER = "stop timer";
+    private static final String START_TIMER = "start timer";
+    private static final int UPDATE_DELAY = 1000; // [ms]
+    private static final int UPDATE_PERIOD = 100; // [ms]
+    private int nSamples = 400;
+    private Timer timer;
+    private TableViewer tableViewer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'tableViewer' could be replaced by a local variable.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4989692)